### PR TITLE
Modify parser for sponsors 

### PR DESCRIPTION
**Pull request by the-dagger**
Modify parser for sponsors to be fetched according to new json outputs from API.
@mananwason Please review and merge if it looks good.
screenshot-area-2016-06-28-170932

### CODE REVIEW

**Comment by the-dagger** [comment]
Failure is due to gradle version.
I'll fix it in a few.

File: gen/org/fossasia/openevent/Manifest.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/*___Generated_by_IDEA___*/
+
+package org.fossasia.openevent;
```
Comment (mananwason) [inline gen/org/fossasia/openevent/Manifest.java]: Autogenerated file. Should be removed or added to gitignore

File: gen/org/fossasia/openevent/R.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/*___Generated_by_IDEA___*/
+
+package org.fossasia.openevent;
+
```
Comment (mananwason) [inline gen/org/fossasia/openevent/R.java]: Autogenerated file. Should be removed or added to gitignore

File: android/app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java [hunk 3]
```diff
@@ -67,9 +68,10 @@ public void onRefresh() {
         sponsorsRecyclerView.addOnItemTouchListener(new RecyclerItemClickListener(view.getContext(), new RecyclerItemClickListener.OnItemClickListener() {
             @Override
             public void onItemClick(View itemView, int position) {
-                String sponsorUrl = dbSingleton.getSponsorList().get(position).getUrl();
-                if (!sponsorUrl.startsWith("http") || !sponsorUrl.startsWith("https")) {
+                String sponsorUrl = dbSingleton.getSponsorList().get(position).getLink();
+                if (!sponsorUrl.startsWith("http")) {
                     sponsorUrl = "http://" + sponsorUrl;
```
Comment (mananwason) [inline android/app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java]: What about https?

File: android/app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java [hunk 4]
```diff
@@ -67,9 +68,10 @@ public void onRefresh() {
...
+                    Log.e("Sponsor URL",sponsorUrl);
```
Comment (mananwason) [inline android/app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java]: Remove Log

**Comment by mananwason** [comment]
@the-dagger I believe this PR focuses on parsing data from OTS but infact we need to parse new json data from the orga server

**Comment by the-dagger** [comment]
@mananwason Isn't that the same as this?
https://github.com/OpenTechSummit/2016.opentechsummit.net/blob/gh-pages/programm/json/

File: android/app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java (hunk 3 above)
Comment (the-dagger) [inline android/app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java]: adding hppts was appending https:// twice before each URL hence making them invalid

**Comment by mananwason** [comment]
No @the-dagger I believe that the output will be different. The server guys must have documented this somewhere. Please coordinate with them. @rafalkowalski @niranjan94 @aviaryan @aditya1702

File: android/app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java (hunk 3 above)
Comment (mananwason) [inline android/app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java]: Why twice? We are checking for https, if not we append

**Comment by niranjan94** [comment]
@the-dagger see https://github.com/fossasia/open-event-orga-server/tree/development/samples/ots16

File: android/app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java (hunk 3 above)
Comment (the-dagger) [inline android/app/src/main/java/org/fossasia/openevent/fragments/SponsorsFragment.java]: Yep i get it, but it was doing this.
Hence as a result all of the links were invalid unless i removed this.

**Comment by the-dagger** [comment]
@niranjan94 Thanks a lot :)

**Pull request closed without merging**